types/chains.go

```diff
@@ -0,0 +1,54 @@
+package types
+
+import "github.com/attestantio/go-eth2-client/spec/phase0"
+
+// DomainType is a unique identifier for signatures, 2 identical pieces of data signed with different domains will result in different sigs
+type DomainType [4]byte
+
+// Chains are intended to separate different SSV chains. A chain can have many forks in it.
+const (
+	MainnetChain = 0x0
+	PrimusChain  = 0x1
+	ShifuChain   = 0x2
+	JatoChain    = 0x3
+)
```

Comment:
I think we should have a convention for consts. For example if mainnet prefix with 0 and if testnet prefix with 1:

```
// mainnet
const (
 MainnetFork0 = 0x00
 MainnetFork1 = 0x01
 MainnetFork2 = 0x02
 )
 
 // testnet (Jato)
 const (
    JatoFork0 = 0x10
    JatoFork1 = 0x11
 )
```